tests/ui/lint/dropping_copy_types-issue-125189-can-not-fixed.rs

```diff
@@ -0,0 +1,17 @@
+//@ check-fail
+
+#![deny(dropping_copy_types)]
+
+fn main() {
+    let y = 1;
+    let z = 2;
+    match y {
+        0 => drop(y), //~ ERROR calls to `std::mem::drop`
+        1 => drop(z), //~ ERROR calls to `std::mem::drop`
+        2 => drop(3), //~ ERROR calls to `std::mem::drop`
+        _ => {
+            drop(y); //~ ERROR calls to `std::mem::drop`
+            drop(z); //~ ERROR calls to `std::mem::drop`
```

Comment:
Nit: these 2 lines don't really fit in with the name of the test, since they can be fixed automatically